src/averageAges.js

```diff
@@ -34,7 +46,26 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let women;
+  if ((typeof withChildren === 'undefined') || (!withChildren)) {
+    women = people.filter((person) => {
+      return person.sex === 'f';
+    });
+  } else {
+    women = people.filter((person) => {
+      if (person.sex !== 'f') return false;
+      return people.some((persona) => {
+        return persona.mother === person.name;
+      });
+    });
+  }
+  const ageSum = women.reduce((sum, person) => {
+    const age = person.died - person.born;
+    return sum + age;
+  }, 0);
+
+  const ageCount = women.length;
+  return (ageSum / ageCount);
```

Comment:
unneeded parenthesis
Do you need `toFixed` in the first function, but not need here?